docs/getting_started/requirements.md

```diff
@@ -291,6 +376,10 @@ Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached,
 
 ## Package manager
 
+=== "Ibexa DXP v4.6"
+
+    - Composer: recent 2.1 version
```

Comment:
Can't we use  the current 2.6.6? Should there be a + here?

Reply:
Note security issue https://blog.packagist.com/composer-2-7-and-cve-2024-24821/ which is only fixed in 2.7 and 2.2.3, it seems.

Reply:
Confirmed, only 2.2.23 and 2.7.0 got the fix, so we should consider how/if we will reflect this in the doc.
https://github.com/composer/composer/security/advisories/GHSA-7c6p-848j-wh5h